src/app/fdctl/configure/ethtool-channels.c

```diff
@@ -9,7 +9,7 @@
 #include <linux/ethtool.h>
 #include <linux/sockios.h>
 
-#define NAME "ethtool"
+#define NAME "ethtool-channels"
```

Comment:
Name of this needs to be updated elsewhere in the book docs and probably some diagnostic comments and things